internal/collect/datacenters.go

```diff
@@ -52,7 +54,14 @@ func Datacenters(config *load.Config) {
 func collectEvents(config *load.Config, d mo.Datacenter, newDatacenter *load.Datacenter) {
 	//https://pubs.vmware.com/vsphere-51/index.jsp?topic=%2Fcom.vmware.wssdk.apiref.doc%2Fvim.HistoryCollector.html
 
-	c := cache.NewCache(d.Name, config.CachePath)
+	path := persist.DefaultPath(config.IntegrationName)
+	store, err := persist.NewFileStore(path, nil, time.Hour*24)
```

Comment:
i think that could be useful to pass the logger in config instead of nil.

Reply:
yeah, makes sense, although I don't really know why the Store needs a logger..

Reply:
I took a look at the Storer code and the logger is not used for anything. I'll still pass in our logger, but it's a noop